src/core/layertreemodel.h

```diff
@@ -112,4 +91,68 @@ class FlatLayerTreeModel : public QAbstractProxyModel
     bool mFrozen = false;
 };
 
+class FlatLayerTreeModel : public QSortFilterProxyModel
+{
+  Q_OBJECT
+
+  Q_PROPERTY( QString mapTheme READ mapTheme WRITE setMapTheme NOTIFY mapThemeChanged )
+
+  public:
+      enum Roles
```

Comment:
Looks like prepare commit was not run?

Reply:
Oups, my home laptop isn't hooked yet. Done.

Reply:
We should add a test for that ...